src/content/app/species/components/species-sidebar-toolstrip/SpeciesSidebarToolstrip.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import noop from 'lodash/noop';
+
+import { useAppSelector, useAppDispatch } from 'src/store';
+import useSpeciesAnalytics from '../../hooks/useSpeciesAnalytics';
+
+import {
+  toggleSidebar,
+  closeSpeciesSidebarModal,
+  openSpeciesSidebarModal
+} from 'src/content/app/species/state/sidebar/speciesSidebarSlice';
+import { SpeciesSidebarModalView } from '../../state/sidebar/speciesSidebarSlice';
```

Comment:
This import doesn't fit in this group

Reply:
This is not a type as `Status`, is it? Please could you tell me why it doesn't fit here?

Reply:
It is actually a `type` since it is an `enum`, though we don't import it as a `type` import.